apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -1787,6 +1787,20 @@ const saveFoldersEpic: AppEpic = (action$, state$) =>
     ignoreElements(),
   );
 
+const hideChatbarEpic: AppEpic = (action$) =>
+  action$.pipe(
+    filter(
+      (action) =>
+        ConversationsActions.uploadConversationsByIdsSuccess.match(action) ||
+        ConversationsActions.saveNewConversationSuccess.match(action) ||
```

Comment:
Maybe it's better to close this on real action but not on success?

Reply:
it's expected to react on `uploadConversationsByIdsSuccess` (create replay, playback, duplicate) and in success case

Reply:
It should react on appropriate actions but not on something which can be reused in another actions

Reply:
create replay, playback, duplicate execute this common action and on this case we hide chatbar